assets/javascripts/kitten/components/box/link-box.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import classNames from 'classnames'
+
+class LinkBox extends React.Component {
+
+  renderIcon() {
+    if (this.props.iconDisplay) {
+      return (
+        <div className="k-LinkBox__icon">
+          <div dangerouslySetInnerHTML={{__html: this.props.iconSrc}} />
```

Comment:
Je ne suis pas fan du `dangerouslySetInnerHTML` car il permet d'injecter du HTML dans notre composant React. 
Est-ce qu'on ne pourrait pas plutôt utiliser `this.props.children` pour mettre ce `svg` dans cette `div` ?

Ce qui impliquerait de créer un élément React container.